File: angular/bootstrap/test/src/app/shared/models/account.ts
export class Account {
  id: string;
  name: string;
  email: string;
  telephone: string;
  fax: string;
  website: string;
  vatNumber: string;
  description: string;
  addressId: string;
  parentAccountId: string;
  billingAccountId: string;
  relationTypeId: string;
  primaryContactId: string;
  createdOn: Date;
  modifiedOn: Date;
  deletedOn: Date;
  createdById: string;
  modifiedById: string;
  address: Address;
  parentAccount: Account;
  billingAccount: Account;
  relationType: AccountRelationType;
  primaryContact: Contact;
  createdBy: User;
  modifiedBy: User;
}

File: dotnet/CodeGen/_Output/Projects/WebAPI/CodeGenOutput.API/Models/Account.cs
using System;
using System.Collections.Generic;

namespace CodeGenOutput.Models
{
	public class Account
	{
		public Guid Id { get; set; }
		public string Name { get; set; }
		public string Description { get; set; }
		public string Telephone { get; set; }
		public string Email { get; set; }
		public string Website { get; set; }
		public string VAT { get; set; }
	}
}
